types/pokemonTypes.go

```diff
@@ -0,0 +1,16 @@
+package pokeTypes
+
+type Pokemon struct {
```

Comment:
This gives me the impression should be inside the model package. Why do you have this `types` package? Its name is ambiguous and not self-descriptive.